server.js

```diff
@@ -18,39 +18,39 @@
 // Task 2 : Connecting with mongo db data base 
 

```

Comment:
Uncomment the line to import the 'MongoClient' from 'mongodb' module as it is required for connecting to MongoDB